### DISCUSSION TITLE
Compatibility issues between different agent versions

### DISCUSSION THREAD

**Discussion by songhao1105** [apache/skywalking#8872] [Q&A] [answered]
Our company now has JDK1.6 and 1.8 projects. We plan to use skywalking6.x agent version for JDK1.6 projects and Skywalking8. x agent version for JDK1.8 projects, and then skywalking8.x version will be used for OAP services. Can I support this? Or how can I adjust the code to support it?

**Comment by wu-sheng** [comment, marked as answer]
JDK 1.6 is not supposed by JDK officially, and we dropped the support one year ago. So, no way to support that from the community.

**Reply by songhao1105** [reply]
If the JDK1.6 project uses sw6.x and the JDK1.8 project uses SW8. x, is there any way to link the two sides?

**Reply by wu-sheng** [reply]
No, official doesn't provide that.

**Reply by songhao1105** [reply]
Does the skywalking6.x support extended metric? If support, could you please give me an guide document?

**Reply by wu-sheng** [reply]
> Does the skywalking6.x support extended metric?

I don't know what you mean. Most metrics are calculated through the OAP server, not the agent if that is your meaning.

6.x were not supported for years. We don't have anything more to provide. The releases are there, and the source codes with documents are there.
